cinema-soft/src/main/java/softserve/academy/cinemasoft/utils/Validator.java

```diff
@@ -8,10 +8,11 @@
 @Component
 public class Validator {
 
+    private final String time12HoursPattern = "([0|1]?[0-9]|2[0-3]):[0-5][0-9]";
```

Comment:
Constant names should be uppercase separated by underscore.

Reply:
Fixed.